CMakeLists.txt

```diff
@@ -70,52 +64,57 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Set the minimum python version to 3.7
+set(PYBIND11_PYTHON_VERSION 3.7)
 
-unset(PYTHON_EXECUTABLE)
-find_package(Python 3.7 COMPONENTS Interpreter Development REQUIRED)
-set(PYTHON_INCLUDE_DIRS ${Python_INCLUDE_DIR})
-set(PYTHON_EXECUTABLE ${Python_EXECUTABLE})
-set(PYBIND11_PYTHON_VERSION ${Python_VERSION})
-find_package(pybind11 QUIET)
-if(NOT pybind11_FOUND)
-  message("-- Cannot build the python module 'Sofa', missing the pybind software to generate the bindings.")
-  return()
-endif()
+# Set the minimum pybind11 version to 2.3 (before that the pybind11::embed target did not exist)
+find_package(pybind11 2.3 CONFIG QUIET REQUIRED)
+
+set(SP3_PYTHON_PACKAGES_DIRECTORY
+    "site-packages"
+    CACHE PATH
+    "Path to the directory where the python packages will be built and installed. (default to site-packages)"
+)
 
+# Get the Python's user site packages directory, or FASLE if not found
+SP3_get_python_user_site()
+CMAKE_DEPENDENT_OPTION(
+    SP3_LINK_TO_USER_SITE
+    "Install python packages to python's user site directory. The directory path can be changed by setting SP3_PYTHON_PACKAGES_LINK_DIRECTORY."
+    ON "PYTHON_USER_SITE" OFF)
+
+set(SP3_PYTHON_PACKAGES_LINK_DIRECTORY
```

Comment:
So If I understand well, there's 2 new options now:
SP3_LINK_TO_USER_SITE -> on / off allows to add symlink to the user's .local/ site
SP3_PYTHON_PACKAGES_LINK_DIRECTORY -> path let's you change the user's local site path

Reply:
Yep, that's exactly it. For example, you could change the `SP3_PYTHON_PACKAGES_LINK_DIRECTORY` variable to `/usr/lib/python3/dist-packages`  to install the bindings for all users (this is what I'm doing for Docker images).